src/conf_mode/interfaces-bridge.py

```diff
@@ -57,6 +60,37 @@ def get_config(config=None):
             bridge['member'].update({'interface_remove': tmp })
         else:
             bridge.update({'member': {'interface_remove': tmp }})
+        
+            
+    # determine which members vlan have been removed
+    
+    if dict_search('member.interface', bridge):
+        for interface in list(bridge['member']['interface']):
+            tmp = leaf_node_changed(conf, ['member', 'interface',interface,'native_vlan'])
```

Comment:
native_vlan and allowed_vlan share the same structure - please combine this in a single helper function which is called with the appropriate key (native_vlan or allowed_vlan)